packages/node_modules/pouchdb-adapter-http/src/index.js

```diff
@@ -237,7 +237,7 @@ function HttpPouch(opts, callback) {
     return setupPromise;
   }
 
-  setTimeout(function () {
```

Comment:
Anywhere where we were calling `setTimeout(fn, 0)` I replaced it with `nextTick()` just to make our code more consistent.